### DISCUSSION TITLE
Problem : CSS works when creating file but not displayed on GitHub page link

### DISCUSSION THREAD

**Discussion by clemence-aimee** [community/community#22360] [New to GitHub]
<p>Hi,<br>
I have a problem when trying to publish my website with a GitHub page, I believe I followed all the correct steps ; only, when I click on the link, first I get a 404 Error and then sometimes when I refresh it, it displays the page but without the CSS</p>
<p>Here’s my project, thanks for your help, I’m new at this :</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/clemence-aimee/landing">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/clemence-aimee/landing" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/95712469/181070528-12b29314-99fa-4116-9759-c3b1b2d77686.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9c4b94d7b0a1ee97bab13203df435f1b7236182e_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9c4b94d7b0a1ee97bab13203df435f1b7236182e_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/c/9c4b94d7b0a1ee97bab13203df435f1b7236182e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9c4b94d7b0a1ee97bab13203df435f1b7236182e_2_10x10.png"></div>

<h3><a href="https://github.com/clemence-aimee/landing" target="_blank" rel="noopener nofollow ugc">GitHub - clemence-aimee/landing</a></h3>

  <p>Contribute to clemence-aimee/landing development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by Katsute** [comment, marked as answer]
<p>The path to your css is wrong, you set it as <code>css/style.css</code> when your file is located at <code>style.css</code>.</p>
<aside class="onebox githubblob" data-onebox-src="https://github.com/clemence-aimee/landing/blob/4c2f622b6475fb49e6b0eab3459604022bcb0e21/index.html#L11">
  <header class="source">

      <a href="https://github.com/clemence-aimee/landing/blob/4c2f622b6475fb49e6b0eab3459604022bcb0e21/index.html#L11" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/clemence-aimee/landing/blob/4c2f622b6475fb49e6b0eab3459604022bcb0e21/index.html#L11" target="_blank" rel="noopener nofollow ugc">clemence-aimee/landing/blob/4c2f622b6475fb49e6b0eab3459604022bcb0e21/index.html#L11</a></h4>



  <pre class="onebox">    <code class="lang-html">
      <ol class="start lines" start="1" style="counter-reset: li-counter 0 ;">
          <li>&lt;!DOCTYPE hmtl&gt;</li>
          <li>
          </li>
<li>
          </li>
<li>  &lt;html&gt;</li>
          <li>
          </li>
<li>
          </li>
<li>  	&lt;head&gt;</li>
          <li>  		&lt;title&gt; Your Very Own Cake&lt;/title&gt;</li>
          <li>  		&lt;meta name="description" content="This is Clemence's profile"&gt;</li>
          <li>  		&lt;meta charset="utf-8"&gt;</li>
          <li class="selected">  		&lt;link rel="stylesheet" type="text/css" href="css/style.css"&gt;</li>
          <li>  	&lt;/head&gt;</li>
          <li>
          </li>
<li> 	&lt;body&gt;</li>
          <li>
          </li>
<li> 		&lt;div class="banner"&gt; </li>
          <li> 			&lt;div class="container"&gt;</li>
          <li> 				&lt;h1&gt;Your Very Own Cake&lt;/h1&gt;</li>
          <li> 				&lt;h2&gt;The best personalized cakes&lt;/h2&gt;</li>
          <li> 				&lt;a href="#start-section" class="btn-blue"&gt;Let's get started&lt;/a&gt;</li>
          <li>			&lt;/div&gt;</li>
      </ol>
    </code>
  </pre>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
